module/product/tests/Domain/Factory/Decorator/CreatedAtAttributeDecoratorTest.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+/**
+ * Copyright © Bold Brand Commerce Sp. z o.o. All rights reserved.
+ * See LICENSE.txt for license details.
+ */
+
+declare(strict_types = 1);
+
+namespace Ergonode\Product\Tests\Domain\Factory\Decorator;
+
+use Ergonode\Category\Domain\ValueObject\CategoryCode;
+use Ergonode\Designer\Domain\Entity\TemplateId;
+use Ergonode\Product\Domain\Entity\ProductId;
+use Ergonode\Product\Domain\Factory\Decorator\CreatedAtAttributeDecorator;
+use Ergonode\Product\Domain\Factory\ProductFactoryInterface;
+use Ergonode\Product\Domain\ValueObject\Sku;
+use Ergonode\Value\Domain\ValueObject\ValueInterface;
+use PHPUnit\Framework\MockObject\MockObject;
+use PHPUnit\Framework\TestCase;
+
+/**
+ */
+class CreatedAtAttributeDecoratorTest extends TestCase
+{
+    /**
+     */
+    public function testDecoratorCreation(): void
```

Comment:
This test i for testing using of Create Method not creation of decorator

Reply:
Also You test that Decorator use create method of decorated service but not test this decorator logic